.github/workflows/golangci-lint.yml

```diff
@@ -26,5 +26,5 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@971e284b6050e8a5849b72094c50ab08da042db8 # v6.1.1
         with:
-          version: v1.59.1
+          version: v1.62.2
```

Comment:
should we just pin to the latest?

Reply:
discussed in community, keep static version